File: frontend/app/models/GeoEvent.ts
type GeoEvent = {
  "id": number,
  "title": string,
  "description": string,
  "eventDate": string | null,
  "latitude": number,
  "longitude": number,
  "imageUrl": string,
  "status": "pending" | "approved",
  "createdBy": string, // username
  "createdAt": Date,
  "updatedAt": Date
}

File: backend/GeoDocs.API/Models/GeoEvent.cs
namespace GeoDocs.API.Models
{
  public class GeoEvent
  {
    public int Id { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public string? EventDate { get; set; }
    public float Latitude { get; set; }
    public float Longitude { get; set; }
    public string ImageUrl { get; set; } = string.Empty;
    public string Status { get; set; } = string.Empty;
    public string CreatedBy { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
  }
}